core/src/components/input/input.ionic.outline.scss

```
@@ -0,0 +1,110 @@
+@import "./input.vars";
+
+// Input Fill: Outline (Ionic Theme)
+// ----------------------------------------------------------------
+
+:host(.input-fill-outline) {
+  --border-radius: 4px;
+  --padding-start: 12px;
+  --padding-end: 12px;
+  --placeholder-color: #535353;
```

This hex is an Ionic design token: `--ionic-color-neutral-600`. I'm not sure if the branch has the latest from https://github.com/ionic-team/ionic-framework/pull/29245. If it does then we should be able to do:
```suggestion
  --placeholder-color: $ionic-color-neutral-600;
```

---

Fixed in https://github.com/ionic-team/ionic-framework/pull/29268/commits/8d3bc577de78188b0839b850a6891e40e9a16929